packages/esm-service-queues-app/src/config-schema.ts

```diff
@@ -258,11 +258,6 @@ export const configSchema = {
             'The header text for the column. Will be translated if it is a valid translation key. If not provided, the header will be based on the columnType.',
           _default: null,
         },
-        headerI18nModule: {
```

Comment:
still need to make corresponding change where `headerI18nModule` is used.

Reply:
Fixed this in the latest commit